vite.config.ts

```diff
@@ -202,6 +202,15 @@ export default defineConfig(({mode}) => {
 							url: '/teams',
 						},
 					],
+					share_target: {
+						action: '/_share-target',
```

Comment:
If that's a route, why not use the vue routing system?

Reply:
This bit is required for the PWA manifest to register the app as a share-target

Reply:
@kolaente, I did change the share_target to use the `GET` method, and now with the processing added to index.ts everthing seems to work as I intended :)

I'll remove the draft status as its ready for review.